Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -577,80 +595,82 @@ public class RestRequest {
                 }
                 let fileManager = FileManager.default
                 try fileManager.moveItem(at: source, to: destination)
-
+                
                 completionHandler(response as? HTTPURLResponse, error)
-
+                
             } catch {
                 completionHandler(nil, RestError.fileManagerError)
             }
         }
         task.resume()
     }
-
+    
     /// Method used by `CircuitBreaker` as the contextCommand
     ///
     /// - Parameter invocation: `Invocation` contains a command argument, Void return type, and a String fallback arguement
-    private func handleInvocation(invocation: Invocation<(Data?, HTTPURLResponse?, Error?) -> Void, Void, String>) {
+    private func handleInvocation(invocation: Invocation<(String), String>) {
         let task = session.dataTask(with: request) { (data, response, error) in
             if error != nil {
-                invocation.notifyFailure()
+                invocation.notifyFailure(error: BreakerError.fastFail)
             } else {
                 invocation.notifySuccess()
             }
-            let callback = invocation.commandArgs
```

Comment:
Why were these deleted? Unless there's something I'm missing, we need to pass the callback to through the invocation and then call it with the datatask result